internal/cel/cel.go

```diff
@@ -294,3 +305,33 @@ func (p *CELParser) CheckStepRunOutAgainstKnownV1(out *StepRunOut, knownType sql
 
 	return nil
 }
+
+func (p *CELParser) EvaluateEventExpression(expr, eventId string, input, additionalMetadata map[string]interface{}) (bool, error) {
```

Comment:
Should be a `cel_test` file, we should add a unit test here

Reply:
sounds good! will give it a shot, haven't written any Go tests yet 🥳